core-bundle/src/DependencyInjection/Security/ContaoLoginFactory.php

```diff
@@ -33,6 +33,12 @@ public function create(ContainerBuilder $container, string $id, array $config, s
 
         $this->createTwoFactorPreparationListener($container, $id);
 
+        // Configure authorization checker to not throw exception if no firewall is active (e.g. on the command line)
+        if ($container->hasDefinition('security.authorization_checker')) {
+            $authorizationChecker = $container->getDefinition('security.authorization_checker');
+            $authorizationChecker->setArgument(3, false);
+        }
```

Comment:
The 3rd argument is controlled by the `security.always_authenticate_before_granting` config, which is `false` by default (and deprecated, as is setting the 3rd and 4th parameter not to `false`). But should we really force this?

Reply:
we're setting the fourth parameter here (starting with `0`), which defaults to `true` and is set to false if you enable the authentication manager (Symfony 6 security).